docs/reference/components/transforms/ansi_stripper.cue

```diff
@@ -7,6 +7,13 @@ components: transforms: ansi_stripper: {
 		Strips [ANSI escape sequences](\(urls.ansi_escape_codes)).
 		"""
 
+	vrl_replacement: {
+		replacement: remap.functions.strip_ansi_escape_codes
+		examples: [
+			#"strip_ansi_escape_codes("\e[46mfoo\e[0m bar")"#
```

Comment:
Should there be an assignment here, or does `strip_ansi_escape_codes` traverse the event?